tests/nightly/nightly-autorun.sh

```diff
@@ -16,7 +16,8 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -f
```

Comment:
This command removes the `nightly_test_secrets.yml` which the executor needs.  Should we move this file to another directory?
I was planning on cloning the vic-internal repo to the home directory so we could have the most updated version of this nightly_test_secrets.yml file on the nightly machine.  Obviously, we would need to point the drone command to execute this file from the right place.